.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [20.x]
```

Comment:
Since this is a public package I think we can go with a  more conservative node version (16 or 18)

Reply:
Done!